### ISSUE TITLE
ImporterConfiguration: Filters

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite-R#712]

**Comment by msevestre** [comment]
What is this??

**Comment by PavelBal** [comment]
I created this issue during our OSMOSES meeting today. I have to check how `DataImporterConfiguration` treats the filters when loading a configuration from XML. I did not consider this use-case.

**Comment by msevestre** [comment]
Aren't you just delegating to ospsuite.core anyways? 
Or are you manipulating the configuration from XML?(hopefully not...)

**Comment by PavelBal** [comment]
No-no, I am not doing anything with xml directly :D But I think there is e.g. no way to display the filters... So I the configuration works as expected, but there is no interface to display the info.

**Comment by msevestre** [comment]
Ok awesome. Makes sense

**Comment by Yuri05** [comment]
Is this issue still valid?

**Comment by PavelBal** [comment]
Yes.
